membership-attribute-service/app/components/TouchpointBackends.scala

```diff
@@ -21,7 +22,7 @@ class TouchpointBackends(
     contactRepositoryOverride: Option[ContactRepository] = None,
     subscriptionServiceOverride: Option[SubscriptionService[Future]] = None,
     zuoraRestServiceOverride: Option[ZuoraRestService] = None,
-    catalogServiceOverride: Option[CatalogService[Future]] = None,
+    catalogServiceOverride: Option[Future[CatalogMap]] = None,
```

Comment:
before we had a CatalogService and basically the only thing we needed was the CatalogMap which goes from ProductRatePlanId to CatalogZuoraPlan (which is the low level catalog model)
All the high level stuff was read in and validated into fully qualified types but never used in the actual code.

So now the whole thing is just the above Map and the CatalogService is just a way of parsing the Json into the Map.